package.json

```diff
@@ -44,10 +45,12 @@
     "snyk-resolve": "^1.0.0",
     "snyk-tree": "^1.0.0",
     "snyk-try-require": "^1.1.1",
-    "then-fs": "^2.0.0"
+    "then-fs": "^2.0.0",
+    "snyk": "^1.319.2"
```

Comment:
I suggest to bump snyk version to v1.320.4 (latest version)
and solve the no newline at the end of `package.json`